src/main/fc/settings.yaml

```diff
@@ -155,6 +155,9 @@ tables:
   - name: nav_overrides_motor_stop
     enum: navOverridesMotorStop_e
     values: ["OFF_ALWAYS", "OFF", "AUTO_ONLY", "ALL_NAV"]
+  - name: osd_plus_code_short
+    # enum: osd_plus_code_short_e
```

Comment:
I forgot to remove this line. Should I push a new commit? Is there a cost to running all the checks for every new push?

Reply:
@avsaase please push new commit